lib/java/com/google/android/material/drawable/DrawableUtils.java

```diff
@@ -293,8 +293,7 @@ public static Drawable compositeTwoLayeredDrawable(
       drawable.setLayerGravity(1, Gravity.CENTER);
     } else {
       Drawable scaledTopLayerDrawable =
-          new ScaledDrawableWrapper(topLayerDrawable, topLayerNewWidth, topLayerNewHeight)
-              .getDrawable();
+          new ScaledDrawableWrapper(topLayerDrawable, topLayerNewWidth, topLayerNewHeight);
```

Comment:
This code is incorrect because [a wrapped drawable](https://developer.android.com/reference/androidx/appcompat/graphics/drawable/DrawableWrapperCompat#getDrawable()) (`topLayerDrawable`) is used instead of a wrapper (scaled `topLayerDrawable`).

We're using `ScaledDrawableWrapper` here to get a scaled `topLayerDrawable` on pre-23 API (on API 23 and above we are using built-in [`LayerDrawable.setLayerSize()`](https://developer.android.com/reference/android/graphics/drawable/LayerDrawable#setLayerSize(int,%20int,%20int)) for this).